Server/VikingVault.Services/TransactionService.cs

```diff
@@ -98,5 +98,38 @@ public List<TransactionDTO> GetTransactions(string userId)
             }
             return transactionsDTO;
         }
+
+        public List<TransactionDTO> GetAllTransactions(string userId)
+        {
+            int uid = int.Parse(userId);
```

Comment:
Please be aware that the `int.Parse` can throw an exception if it cannot parse the given string. (Just in case you want to catch this somewhere I'm saying)

Reply:
Understood, I've changed it. Thank you!